src/parser/statement.js

```diff
@@ -632,8 +632,14 @@ export default class StatementParser extends ExpressionParser {
   }
 
   parseFunctionParams(node: N.NormalFunction): void {
+    node.params = this.doParseFunctionParams();
+  }
+
+  doParseFunctionParams(): $ReadOnlyArray<N.Pattern> {
     this.expect(tt.parenL);
-    node.params = this.parseBindingList(tt.parenR);
+    // $FlowIgnore (won't be TSParameterProperty)
+    const x: $ReadOnlyArray<N.Pattern> = this.parseBindingList(tt.parenR);
+    return x;
```

Comment:
I needed that specific functionality from within the typescript plugin.
But it looks like it's simpler if I just call `super.parseFunctionParams` to access that instead.